pkg/flagutil/github_enablement.go

```diff
@@ -81,14 +81,14 @@ func (o *GitHubEnablementOptions) EnablementChecker() func(org, repo string) boo
 	enabledOrgs := o.enabledOrgs.StringSet()
 	enabledRepos := o.enabledRepos.StringSet()
 	disabledOrgs := o.disabledOrgs.StringSet()
-	diabledRepos := o.disabledRepos.StringSet()
+	disabledRepos := o.disabledRepos.StringSet()
```

Comment:
we can probably remove both

Reply:
Can we? The flag doesn't mention that the functionality is deprecated.

In any case I'd do that in a dedicated PR :)